src/HexUtils.js

```diff
@@ -40,6 +40,15 @@ class HexUtils {
     return HexUtils.add(hex, HexUtils.direction(direction));
   }
 
+  static neighbours(hex) {
```

Comment:
ahhh! yeah that would be super nice. I am using it to find hexes on the border in order to build oceans around my island. So I check a hex's neighbors and if any are not in the rendered map it is a border hex. But it would be easier to just get rendered hexes because them I could count what was returned, and if it is less than 6 I know I am on an edge.

![screen shot 2017-05-02 at 10 05 05 am](https://cloud.githubusercontent.com/assets/925980/25624410/e33e2122-2f1e-11e7-9072-eb3f92d826d1.png)